File: webservices/src/main/java/fieg/modulos/cadastro/conveniobancario/dto/FaixaNossoNumeroDTO.java
package fieg.modulos.cadastro.conveniobancario.dto;

import lombok.Data;

import java.time.LocalDateTime;

@Data
public class FaixaNossoNumeroDTO {

    private Long id;
    private Boolean ativo;
    private String nossoNumeroInicial;
    private String nossoNumeroFinal;
    private String nossoNumeroAtual;
    private LocalDateTime dataInclusao;
    private LocalDateTime dataAlteracao;
    private Integer idOperadorAlteracao;
    private Integer idOperadorInclusao;
}

File: web/src/models/DTOs/ConvenioBancario/FaixaNossoNumeroDTO.ts
export interface FaixaNossoNumeroDTO {
  id: number;
  ativo: boolean;
  nossoNumeroInicial: string;
  nossoNumeroFinal: string;
  nossoNumeroAtual: string;
  dataInclusao: Date;
  dataAlteracao: Date | null;
  idOperadorAlteracao: number | null;
  idOperadorInclusao: number;
}
